collections/brandedApplicantConversations.js

```diff
@@ -9,7 +9,7 @@ BrandedApplicantConversations.attachSchema(new SimpleSchema({
   messages: {type: [Object]},
   handled: {type: Boolean, optional: true, defaultValue: false},
   'messages.$.body': {type: String},
-  'messages.$.prefix': {type: String},
+  'messages.$.prefix': {type: String, optional: true}, // Only applies to escalations from Mascot/Phoenix
```

Comment:
So what it does *not* apply to are responses from the college, right? Otherwise, the message should have a prefix. I would find it more helpful to say what this does not apply to, rather than what it does, as I think of escalations as the primary case of messages.

Reply:
Hmm... apparently I don't completely understand this collection; I was thinking that messages from the student were also in it, but that would be a little redundant, even by our standards.  The actual change in https://github.com/AdmitHub/NeOliTh/pull/908 just extends the current logging to log the prefix separately from the body wherever it's already being passed in, which I assume is just the escalation.

If the response is the only other thing in here, then I'd be fine inverting the comment to say it's not logged on responses.

Reply:
I believe so -- I think the only thing in here is escalated messages, plus the response from the college officer, plus the student's response to the response, and so on, until the conversation ends. I *think* only the first of those has a prefix, though my confidence is not 100%.

Reply:
On second thought, it seems useful to include what it is when it exists, so left that in and just added a sentence on.  Good enough?